src/robot/model/tags.py

```diff
@@ -63,6 +63,19 @@ def __unicode__(self):
     def __repr__(self):
         return repr(list(self))
 
+    def __eq__(self, other):
+        if type(other) is not Tags:
+            return NotImplemented
+
+        # noinspection PyProtectedMember
```

Comment:
JetBrains/PyCharm linter

Reply:
@pekkaklarck 
> Anyway, because we don't have standard IDE/linter for our project, it's not good to add magic comments that only work with sone isn't, in my opinion, a good idea.